### PR TITLE
fix some comments

### PR DESCRIPTION
**Pull request by fate-grand-order**
Signed-off-by: fate-grand-order <chenjg@harmonycloud.cn>

### CODE REVIEW

File: cluster/cluster.go [hunk 1]
```diff
@@ -10,100 +10,100 @@ import (
...
-	// Return all containers
+	// Containers returns all containers.
 	Containers() Containers
 
-	// Start a container
+	// StartContainer starts a container.
 	StartContainer(container *Container, hostConfig *dockerclient.HostConfig) error
 
-	// Return container the matching `IDOrName`
+	// Container returns container the matching `IDOrName`.
```
Comment (allencloud) [inline cluster/cluster.go]: I do not think this line is readable.
Could you help to make it better?

Comment (fate-grand-order) [inline cluster/cluster.go]: ok，I will update it.

**Comment by fate-grand-order** [comment]
I have fixed the comment in cluster/cluster.go, please review it again. @allencloud

**Comment by dongluochen** [comment]
LGTM

**Pull request merged**